### DISCUSSION TITLE
Welcome to ical4j Discussions!

### DISCUSSION THREAD

**Discussion by benfortuna** [ical4j/ical4j#471] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Curious-Victor** [comment]
Hi there,

Thanks for creating and maintaining this project.

I'm not a developer, but I've noticed that the Bintray Releases link seems broken (https://bintray.com/ical4j/maven/ical4j) and maybe you would like to update it.

Kind Regards,

Victor

**Reply by benfortuna** [reply]
Hi Victor,

Yes bintray stopped support for free release management, so I've been in the process of removing all references to JFrog/Bintray.

If you do see anywhere in the code still using it let me know and I can remove (I now deploy directly to maven central instead - https://search.maven.org)

**Reply by Curious-Victor** [reply]
Thank you very much for the clarification and assistance Ben.

It is much appreciated.
